test/e2e/storage/testsuites/snapshotsecrets.go

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
The snapshot secrets test should not be added as a file under testsuites.  You need to add a new test for secrets in test/e2e/storage/csi_mock_volume.go, similar to what is done in this PR:

https://github.com/kubernetes/kubernetes/pull/95863/files#diff-ffda3f0c41f167e7deedad89b23cf9b89d0e9805f910c81e075baa19d667f21fR1179